src/views/QuestionBank/js/question-bank-detail.js

```diff
@@ -53,37 +64,115 @@ export default {
     failFetchingQuestionBankDetail () {
       this.$toasted.error('Something went wrong')
     },
-    cancelButtonClicked () {
-      if (this.editMode) {
-        this.initPage()
-        this.editMode = !this.editMode
-        return
+    getQuestionList($state) {
+      this.state = $state
+      this.fetchQuestionBankQuestionList({
+        data: {
+          bankId: this.$route.params.bankId,
+          page: this.paging.page,
+          size: this.paging.size
+        },
+        callback: this.successFetchingQuestionBankQuestionList,
+        fail: this.failFetchingQuestionBankQuestionList
+      })
+    },
+    successFetchingQuestionBankQuestionList (paging, response) {
+      this.paging = paging
+      if (response.length) {
+        this.questions.push(...response)
+        this.paging.page++
+        this.state.loaded()
+      } else {
+        this.state.complete()
       }
+    },
+    failFetchingQuestionBankQuestionList () {
+      this.$toasted.error('Something went wrong')
```

Comment:
create issue to implement error handling

Reply:
Already made, refer to issue #160